src/utils/key.rs

```diff
@@ -179,6 +179,22 @@ impl KeyBundle {
 
                 Ok(encrypt(cipher, &self.key, None, data)?)
             }
+            "rsa" => {
+                let rsa = Rsa::private_key_from_pem(&self.key)?;
+                if data.len() >= rsa.size() as usize {
```

Comment:
IIRC, data length can be equal to the key length. It's okay with the RSA algorithm.

Reply:
> IIRC, data length can be equal to the key length. It's okay with the RSA algorithm.

fixed